tests/js/PrettierLintBearTest.py

```diff
@@ -14,6 +14,38 @@
 """
 # Stop ignoring
 
+bad_semicolon = """
+var x;
+"""
+
+good_quotes = """const a = () => "Foo bar";
+
+function b(object, key) {
+  return object["key"];
+}"""
+
+bad_quotes = """const a = () => 'Foo bar';"""
+
+bad_paranethesis = """function b(){
+"""
+
+good_paranethesis = """function b() {}
+"""
+
+bad_indent = """if (x) {
+x = 0
+}"""
+
+good_indent = """if (x) {
+  x = 0;
+}"""
+
 PrettierLintBear = verify_local_bear(PrettierLintBear,
-                                     valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     valid_files=(good_quotes,
+                                                  good_file,
+                                                  good_paranethesis,
```

Comment:
'good_paranethesis' is wrong spelling. (same issue for 'bad_paranethesis')

Reply:
thanks!